File: java/src/main/java/com/iteso/nintendo/attacks/FireFang.java
package com.iteso.nintendo.attacks;

import com.iteso.nintendo.attacks.Attack;

public class FireFang implements Attack {
    static int damage;
    static String name = "Fire Fang";

    @Override
    public String attackName() {
        return name;
    }

    @Override
    public int attackDamage() {
        return this.damage;
    }

    @Override
    public void setDamage(int damage) {
        this.damage = damage;
    }
}

File: python/FireFang.py
class FireFang:
    def __init__(self):
        self.damage = None
        self.name = 'Fire Fang'

    def attackName(self):
        return self.name

    def attackDamage(self):
        return self.damage

    def setDamage(self, damage):
        self.damage = damage
